From: Steve Lhomme <robux4@ycbcr.xyz>
Subject: [PATCH 2/2] widl: open files to write in binary mode
Message-Id: <20201216091429.18075-2-robux4@ycbcr.xyz>
Date: Wed, 16 Dec 2020 10:14:28 +0100
In-Reply-To: <20201216091429.18075-1-robux4@ycbcr.xyz>
References: <20201216091429.18075-1-robux4@ycbcr.xyz>

When compiled with mingw64 the output files have Windows line endings instead
of UNIX ones.
---
 tools/widl/header.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/widl/header.c b/tools/widl/header.c
index 43be8d74cbe..5457185538e 100644
--- a/tools/widl/header.c
+++ b/tools/widl/header.c
@@ -1893,7 +1893,7 @@ void write_header(const statement_list_t *stmts)
 
   if (!do_header) return;
 
-  if(!(header = fopen(header_name, "w"))) {
+  if(!(header = fopen(header_name, "wb"))) {
     error("Could not open %s for output\n", header_name);
     return;
   }

-- 
2.29.2

